diffkemp/simpll/passes/FunctionAbstractionsGenerator.h

```diff
@@ -18,6 +18,9 @@
 #include <llvm/IR/PassManager.h>
 #include <set>
 
+const std::string inlineAsmPrefix = "simpll__inlineasm_";
+const std::string indirectFunctionPrefix = "simpll__indirect_";
```

Comment:
Use better names to express that this is a SimpLL-specific constant:
SimpllInlineAsmPrefix and SimpllIndirectFunctionPrefix.